src/Core/src/Extensions/EnumerableExtensions.cs

```diff
@@ -20,6 +20,29 @@ public static void ForEach<T>(this IEnumerable<T> enumeration, Action<T> action)
 			}
 		}
 
+		/// <summary>
+		/// 
```

Comment:
Empty documentation

Reply:
This is reverting the already existing code.